src/main/java/somethingrandom/entity/ActionableItem.java

```diff
@@ -13,55 +13,43 @@ public class ActionableItem extends Item {
     /**
      * The Name.
      */
-    public String name;
+    private String name;
     /**
      * The ID.
      */
-    public final UUID id;
+    private final UUID id;
     /**
      * The Creation date.
      */
 
-    public final Instant creationDate;
+    private final Instant creationDate;
     /**
      * The amount of time needed to complete the task.
      */
     @Nullable
-    public Duration neededTime;
-    /**
-     * The Item kind.
-     */
-    public final ItemKind itemKind;
+    private Duration neededTime;
 
     /**
      * Instantiates a new Actionable item.
      *
      * @param name         the name
      * @param creationDate the creation date
      * @param neededTime   the needed time
-     * @param itemKind     the item kind
      */
-    public ActionableItem(String name, Instant creationDate, @Nullable Duration neededTime, ItemKind itemKind){
+    public ActionableItem(String name, Instant creationDate, @Nullable Duration neededTime){
```

Comment:
nitpick: probably should be space between `)` and `{`

Reply:
Also, it'd be nice if there was a blank line between each method (before the documentation if applicable), but I can live without it.